src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession cinemaHall);
```

Comment:
The parameter name should reflect that it's a `MovieSession` object, not a `cinemaHall`. Consider renaming it to `movieSession` to avoid confusion and to be consistent with the entity it represents.